### PR TITLE
Fix zero address ens name

### PR DESCRIPTION
**Pull request by nebolax**
Closes: #4657

### CODE REVIEW

File: rotkehlchen/chain/ethereum/manager.py [hunk 1]
```diff
@@ -658,7 +658,11 @@ def ens_reverse_lookup(self, reversed_addresses: List[ChecksumEthAddress]) -> Di
 
         # Processing human names query output
         for addr, params, human_name_output in zip(filtered_reversed_addresses, human_names_params, human_names_output):  # noqa: E501
-            human_names[addr] = _decode_ens_contract(params=params, result_encoded=human_name_output)  # noqa: E501
+            decoded_name = _decode_ens_contract(params=params, result_encoded=human_name_output)
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/manager.py:661]: This won't work only here. You also call `_decode_ens_contract` above in line 635 and the same treatment should happen there.

But instead what would make sense is to make `_decode_ens_contract` check if it's the zero address and only then return `None`. So change return type of the function too.

**Review by LefterisJP — COMMENTED** [review]
I left a comment. But this also does not answer why is this happening only for this address?

This can't be the only address we have ever seen without a reverse resolver properly set up. So far everything else was working fine.

Also add a changelog entry for this bug: "Fix an edge case at ENS reverse resolution making an address resolving to the zero address instead of a name".

File: rotkehlchen/chain/ethereum/manager.py (hunk 1 above)
Comment (nebolax) [reply rotkehlchen/chain/ethereum/manager.py:661]: Yeah okay

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**